app/projects/page.tsx

```diff
@@ -23,9 +26,31 @@ export const metadata: Metadata = {
 };
 
 const ProjectsPage = () => {
+  const exampleProjectCardData: ProjectCardProps = {
```

Comment:
Importing `ProjectCardProps` is redundant because the TS can infer the object type

Reply:
If i remove `ProjectCardProps` from the object type everything works just fine...
Of course it's not a mistake just feel that it's redundant.
If you disagree feel free to resolve this conversion and ignore my suggestion.

Reply:
```ProjectCardProps``` is there so the benefit of having a type happens as close as possible to the object
- Missing a field that is required by the ```ProjectCardProps``` will give an error
- Adding a field that isn't on the ```ProjectCardProps``` will give an error
- Give intellisense when adding fields to the object

This is a nice discussion so let's go through it :)

By the logic you outlined the ```Metadata``` type that exists in the same page can be removed from ```metadata: Metadata```
If you remove any type everything on the site works just fine
In that sense types **are** redundant